example/src/main/java/io/netty/example/http2/tiles/Launcher.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2015 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.netty.example.http2.tiles;
+
+import io.netty.example.http2.tiles.http.HttpServer;
+import io.netty.example.http2.tiles.http2.Http2Server;
+
+/**
+ * Launches both Http and Http2 servers using Netty to display a set of images
+ * and simulate latency. It is a Netty version of the <a
+ * href="https://http2.golang.org/gophertiles?latency=0"> Go lang HTTP2 tiles
+ * demo</a>.
+ *
+ * Please note that you MUST use JDK 1.8 to run this example and have the
```

Comment:
Also the ALPN library does not require JDK 1.8. It can work with 1.7.  I believe it is just that JDK 1.8 is the only JDK that has ciphers that are not on HTTP/2's black list.